### ISSUE TITLE
WowProgress not working for italian realm because of localized name

### ISSUE THREAD

**Issue by FabryB** [greatman/legendarybot#22]
Hello,
I started using your bot on our discord server, works very well so far, so first of all thank you for your work :)
The issue with WowProgress we are facing is because of italian translation of our realm name:
Italian name -> pozzo dell'eternità
English name -> well of eternity
This is our guild armory link:
http://eu.battle.net/wow/en/guild/pozzo-delleternita/Adversus/
and instead this is our guild link within WowProgress:
https://www.wowprogress.com/guild/eu/well-of-eternity/Adversus

So I think there should be a mechanism which uses english realm name when querying WP instead of localized name.
Don't know if this is affecting other realms beside the Italian one

**TL;DR**
For some reason WowProgress is using english name "well of eternity" for "pozzo dell'eternità" realm. Need to switch realm name when making queries to WP

**Comment by greatman** [comment]
Thanks for the report, ill look to fix the problem.

**Comment by greatman** [comment]
I switched the !wprank command to use Raider.IO. At least they allows localisation of realms. Also, the ranking is really close between the 2 sites. Is also now gives all the raids rankings.
